docs/sources/getting-started/timeseries-dimensions.md

```diff
@@ -8,9 +8,9 @@ weight = 600
 
 # Time series dimensions
 
-At the end of the ["Time series databases" section of "Introduction to time series"]({{< relref "timeseries.md#time-series-databases" >}}), the concept of _labels_, also called _tags_, is introduced:
+In [Introduction to time series]({{< relref "timeseries.md#time-series-databases" >}}), the concept of _labels_, also called _tags_, is introduced:
```

Comment:
Did not find any reference to `_labels_` or `_tags_` in the topic: Introduction to time series. FYI.

Reply:
@kylebrandt, are you sure this sentence is correct? Should we remove it?

Reply:
It is there:

![image](https://user-images.githubusercontent.com/1692624/105093665-1b414a00-5a71-11eb-98e9-8ebdf727e868.png)

![image](https://user-images.githubusercontent.com/1692624/105093733-3318ce00-5a71-11eb-8bfa-142db3dd66f9.png)

that ... table? doesn't look so well :-P

Reply:
Ok. I searched using "_labels_" and "_tags_", that is why there was no result for me.